Doc/reference/datamodel.rst

```diff
@@ -1578,8 +1578,8 @@ Class Binding
    ``A.__dict__['x'].__get__(None, A)``.
 
 Super Binding
-   If ``a`` is an instance of :class:`super`, then the binding ``super(B,
-   obj).m()`` searches ``obj.__class__.__mro__`` for the base class ``A``
+   If ``a`` is an instance of :class:`super`, then the binding ``super().m()``
+   searches ``obj.__class__.__mro__`` for the base class ``A``
    immediately preceding ``B`` and then invokes the descriptor with the call:
```

Comment:
Hmm, this doesn't sound right, as `super()` looks at the base classes *following* the given class, not preceding it. It also doesn't necessarily use the *immediately* following class in the MRO, but rather the first one that defines the attribute of interest.